### ISSUE TITLE
Edytowanie ceny i ilośći wielu ofert jednocześnie

### ISSUE THREAD

**Issue by request4t** [allegro/allegro-api#10097]
Witam!

Czy można edytować ceny i ilość wielu ofert jednocześnie? Każda oferta w requeste ma swoją cenę i ilość.

**Comment by Lukasz-Zurek** [comment]
Nie, takiej możliwości w naszym API nie ma. Możliwa jest tylko [edycja wielu ofert jednocześnie](https://developer.allegro.pl/tutorials/jak-zarzadzac-ofertami-7GzB2L37ase#edycja-wielu-ofert-jednoczesnie), ale przy założeniu, że zmiana obejmuje tę samą cenę, lub liczbę sztuk.

**Comment by request4t** [comment]
Dziękuję za odpowiedź. Teraz już wiem na pewno)
